libs/core/src/lib/carousel/carousel.component.ts

```diff
@@ -391,6 +394,19 @@ export class CarouselComponent implements OnInit, AfterContentInit, AfterViewIni
         }
     }
 
+    private _onSlideUpdates(): void {
```

Comment:
Can you add `/** @hidden */` comment there?

Reply:
added.